README.adoc

```diff
@@ -2,6 +2,8 @@
 
 = Coverage pool
 
+https://github.com/keep-network/coverage-pools/actions/workflows/contracts.yml[image:https://img.shields.io/github/workflow/status/keep-network/coverage-pools/Solidity/main?event=push&label=Coverage%20pool%20contracts%20build[Coverage pool contracts build status]]
```

Comment:
Why not `Solidity tests` or `Solidity build`?

Reply:
OK, I see this is aligned with the convention in https://github.com/keep-network/keep-core/pull/3398.